packages/frontend/package.json

```diff
@@ -7,47 +7,49 @@
     "build": "NODE_ENV=production next build",
     "start": "NODE_ENV=production next start",
     "type-check": "tsc --pretty --noEmit",
-    "format": "prettier --write .",
-    "lint": "eslint --ext .js,.ts,.tsx --fix .",
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx --fix .",
+    "format": "pnpm lint && prettier --write .",
```

Comment:
I'd be in favour of keeping `format` as a prettier script (no lint).

In turn I'd suggest we create a validate script (or maybe a turbo pipeline?) called validate, which aggregates linting, formatting, type-checking and testing in the appropriate order. Then we know validate is the script we run before committing or pushing.

What do you think?

Reply:
i run those commands manually very rarlely anyways 🤷‍♂️ or do you mean by "is the script we run before committing or pushing" that we configure lint-staged to run something like `pnpm turbo run validate`?

Reply:
So my first point is that I don't like running lint inside a script called format. It's not a big deal, it's just that I would have chosen a different name. How about "fix"? Also are we ever going to run lint without wanting to run prettier write afterwards? If so, maybe we could just merge both into a script called fix?

My second point is that I usually like having a "validate" script that runs a bunch of code quality validations: type checking, lint, formatting, tests and build. But I think we can take that discussion out of this PR. I expect it will surface once we start adding tests.